Added and fixed some item_equipment.sql entries.

helixhamin opened the pull request:
Added mods for those also (and a few that were missing that I spotted).
Please note, the changes to item_latents.sql is just moving things around so they are laid out more logically. I limited it to the first 200ish lines so as not to overwhelm a reviewer.

helixhamin commented:
Added/fixed up to 21746.
More will be added after this (to prevent changes from being viewable in browser...you already have to expand just to see the changes xD )

TeoTwawki commented on sql/item_mods.sql:
```diff
@@ -26606,46 +26677,86 @@ INSERT INTO `item_mods` VALUES (21707,2,78);      -- barbarity_+1 HP+78
+INSERT INTO `item_mods` VALUES (21722,30,40);     -- Dolichenus: Magic Accuracy+40
+INSERT INTO `item_mods` VALUES (21722,311,217);   -- Dolichenus: Magic Damage+217
+INSERT INTO `item_mods` VALUES (21722,642,120);   -- Dolichenus: "Decimation" damage +120% (The "Decimation" damage boost applies to all hits in the Weapon Skill.)
+-- INSERT INTO `item_mods` VALUES (21722,??,72);  -- Dolichenus: Main hand: "Decimation" (Grants access to Decimation regardless of skill level or completion of Axe the Competition; while equipped in the main hand only.)(This one may need to be a latent, but no latent exists for equipped in main hand)
```
ok this is getting out of hand. once it makes me left/right scroll on a 1900 wide screen, the comment is too dang long.

helixhamin replied:
Okay, I can shorten it tomorrow. Just trying to make sure it has information needed for someone to be able to implement.

helixhamin commented:
@TeoTwawki Okay, I removed the extra comments. The information is in the wikis, so guess we can look it up when we are ready to implement.

helixhamin commented:
I have also fixed some item mistakes in item_usable.sql and some naming issues with some of the items, including renaming any scripts for the corrected item names if they had scripts. Done up to item_usable item 5716. Items after this may still have existing errors.
There are lots of missing items in item_usable table too...

helixhamin commented:
Okay, does anyone know what Travis is complaining about?

helixhamin commented:
sigh, okay, I know the issue.
It appears that the python price checker command does not like comments in sql files. So, if an npc sells an item, and you add a comment after the item_basic.sql entry for that item, it will cause the python buyer to freak out and exit...
It appears removing the comments that I added for reviewers to more easily check had to be deleted. Sorry...

TeoTwawki commented on sql/item_usable.sql:
```diff
@@ -471,9 +471,9 @@ INSERT INTO `item_usable` VALUES (4525,'pumpkin_pie_+1',1,1,26,0,0,0,0,0);
-INSERT INTO `item_usable` VALUES (4530,'rolanberry_(874_ce)',1,1,26,0,0,0,0,0);
-INSERT INTO `item_usable` VALUES (4531,'rolanberry_(864_ce)',1,1,26,0,0,0,0,0);
+INSERT INTO `item_usable` VALUES (4529,'rolanberry_(881_c.e.)',1,1,26,0,0,0,0,0);
+INSERT INTO `item_usable` VALUES (4530,'rolanberry_(874_c.e.)',1,1,26,0,0,0,0,0);
```
no, we stripped those on purpose.

helixhamin commented:
@TeoTwawki Okay, reverted the name changes that included the punctuation in the name. Rest should all be appropriate now (saute, not sautee, etc)

TeoTwawki commented on sql/item_basic.sql:
```diff
@@ -5327,8 +5327,8 @@ INSERT INTO `item_basic` VALUES (5672,0,'dried_berry','dried_berry',12,1548,57,0
 INSERT INTO `item_basic` VALUES (5675,0,'withered_rolanberry','withered_berry',1,58968,0,0,4624);
-INSERT INTO `item_basic` VALUES (5676,0,'serving_of_mushroom_sautee','mushroom_sautee',1,1580,54,0,764);
-INSERT INTO `item_basic` VALUES (5677,0,'serving_of_patriarch_sautee','patriarch_sautee',1,1580,54,0,955);
+INSERT INTO `item_basic` VALUES (5676,0,'serving_of_mushroom_saute','mushroom_saute',1,1580,54,0,4025);
```
4025 on the NQ and 955 on the HQ?
#shitWikiSays

One or both of them is wrong, and I wouldn't put it past Joe random wiki editor to have put the price he sold it to an guid NPC for instead of a regular shop price. I'd rather keep a lower price unless I see retail confirmation on them both.

helixhamin replied:
Hmm, well, according to ffxiah:

> Mushroom: NPC ~ 4025 Gil
So I changed that one to match.
I checked ffxicyclopedia, for the HQ it says: Resale Price: 4,912~5,022 gil
For NQ, it says: Resale Price: 4,025~4,125 gil

I will adjust (since I didn't actually touch the HQ price since I did not see a price immediately, and didn't notice until you mentioned xD )

TeoTwawki replied:
I saw both of wiki's pages on it..And I don't trust what I see there in ffxiclopedia. The same SE that makes nearly everything sub1 is gonna put this random food between 4 and 5k? seems unlikely doesn't it? And sicne we normally add items with an unknown price at 0, that begs the question of where our current values came from - I haven't looked by in the history but maybe somebody checked this before.

helixhamin commented:
@TeoTwawki Okay, updated. In this case, the first price I noted on ffxiah, not cyclopedia, so I assume they are pretty close to accurate. I adjusted the HQ version to match what cyclopedia states, since no one commented on it on ffxiah.

TeoTwawki commented:
Person of ffxiah is likely just copy pasting from clopedia. A lot of ppl will do that.

This revision _from 2014_ shows a source where the number came from:

https://ffxiclopedia.fandom.com/wiki/Mushroom_Saute?diff=prev&oldid=1478999

But given the age, I could see it having been changed. SE nerfed the hell out of a lot of NPC prices over the years. If free week weren't over I'd just go craft one and sell it to see. If you wanna ask the capture room to go screenshot it, hold off on changing it back and we can be sure.

TeoTwawki commented:
so looks like from Nyu's screenshot in discord, it starts out closer to 3900 but its still actually pretty high.
